spec/rails_helper.rb

```diff
@@ -1,50 +1,39 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
-require 'spec_helper'
-require 'rspec/rails'
-require 'simplecov'
-SimpleCov.start 'rails'
-# Add additional requires below this line. Rails is not loaded until this point!
+require "spec_helper"
+require "rspec/rails"
+require "capybara/rspec"
+require "capybara/rails"
+require "capybara/poltergeist"
+require "capybara-screenshot"
+require "capybara-screenshot/rspec"
+require "simplecov"
+SimpleCov.start "rails"
 
-# Requires supporting ruby files with custom matchers and macros, etc, in
-# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
-# run as spec files by default. This means that files in spec/support that end
-# in _spec.rb will both be required and run as specs, causing the specs to be
-# run twice. It is recommended that you do not name files matching this glob to
-# end with _spec.rb. You can configure this pattern with the --pattern
-# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
-#
-# The following line is provided for convenience purposes. It has the downside
-# of increasing the boot-up time by auto-requiring all files in the support
-# directory. Alternatively, in the individual `*_spec.rb` files, manually
-# require only the support files necessary.
-
-Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
 RSpec.configure do |config|
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
-  # RSpec Rails can automatically mix in different behaviours to your tests
-  # based on their file location, for example enabling you to call `get` and
-  # `post` in specs under `spec/controllers`.
-  #
-  # You can disable this behaviour by removing the line below, and instead
-  # explicitly tag your specs with their type, e.g.:
-  #
-  #     RSpec.describe UsersController, :type => :controller do
-  #       # ...
-  #     end
-  #
-  # The different available types are documented in the features, such as in
-  # https://relishapp.com/rspec/rspec-rails/docs
-  config.infer_spec_type_from_file_location!
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app,
+                                      js_errors: false,
+                                      phantomjs_options: ["--proxy-type=none"],
+                                      timeout: 180,
+                                      :phantomjs_logger => File.open("#{Rails.root}/log/test_phantomjs.log", "a"),
+                                      extensions: [])
+  end
 
-  # Filter lines from Rails gems in backtraces.
+  Capybara.javascript_driver = :poltergeist
+  Capybara.default_selector = :css
+  Capybara.default_wait_time = 5
+
+  Capybara::Screenshot.register_filename_prefix_formatter(:rspec) do |example|
```

Comment:
Unused block argument - `example`. You can omit the argument if you don't care about it.